jansson.pxd

```diff
@@ -4,26 +4,9 @@ cdef extern from "jansson.h":
     ctypedef struct json_t:
         pass
 
-    ctypedef struct json_auto_t:
-        pass
-
     ctypedef struct json_error_t:
-        int line
-        int column
-        int position
-        char *source
-        char *text
-
-    json_t *json_pack(const char *fmt, ...)
-    json_t *json_pack_ex(json_error_t *error, size_t flags, const char *fmt, ...)
-
-    int json_unpack(json_t *root, const char *fmt, ...)
-    int json_unpack_ex(json_t *root, json_error_t *error, size_t flags, const char *fmt, ...)
+        pass
```

Comment:
Please leave the fields here. I like to use them later to implement proper error reporting.